kwanyong/app.js

```diff
@@ -0,0 +1,53 @@
+require("dotenv").config();     
+
+const express = require('express');
+const logger = require('morgan');
+const cors = require('cors');
+const { DataSource } = require('typeorm');
+
+
+const appDataSource = new DataSource({
+    type: process.env.DB_CONNECTION,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE
+})
+
+appDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    })
+
+const app = express()
+ 
+app.use(cors());                  
+app.use(logger('combined'));
+app.use(express.json());
+
+const port = process.env.PORT
+
+app.get('/ping', function (req, res, next) {   
+  res.json({message: 'userCreated'})
+})
+
+app.post('/users', async (req, res) => {
+	const { name, email, profileImage, password} = req.body
+    
+	await appDataSource.query(
+		`INSERT INTO users(
+		    name,
+		    email,
+		    profile_image,
+        password
+		) VALUES (?, ?, ?, ?);
+		`,
+		[ name, email, profileImage, password ]
+	); 
+     res.status(201).json({ message : "successfully created" });
+	})
+ 
+app.listen(port, function () {
+  console.log('server listening on port 8000')
```

Comment:
port는 환경변수로 관리하고 있기 때문에 어떤 port를 사용하고 있는 지에 대한 정보도 변수로 사용할 수 있습니다.

8000이라는 값을 직접 드러내지 않을 수 있는 두가지 방법을 참고해보세요!

```suggestion
// 일반 변수 사용
console.log('server listening on port', port);

// template literal
console.log(`server listening on port ${port}`);
```